README.md

```diff
@@ -125,7 +125,6 @@ WARNING: You have not specified enough hosts to satisfy the blueprint.
 ```
 $ yaas cluster create cluster1 --template=template1.json
 $ cat template2.json | yaas cluster create cluster2
-$ yaas cluster create cluster3
```

Comment:
I think we already support that functionality. You still need to upload a sort of cluster creation template. We could document that this is possible, but I don't think we need any code to support this.